src/licensedcode/data/licenses/any-osi.LICENSE

```diff
@@ -0,0 +1,16 @@
+---
+key: any-osi
+short_name: Any OSI License
+name: Any OSI License
+category: Unstated License
+owner: Unstated
+spdx_license_key: any-OSI
+other_urls:
+    - https://metacpan.org/pod/Exporter::Tidy#LICENSE
+ignorable_urls:
+    - http://www.opensource.org/licenses/alphabetical
+---
+
```

Comment:
Why always adding an empty line there? I reckon it is stripped when loading, but why having this?

Reply:
> Why always adding an empty line there?

I think we had an issue in https://github.com/nexB/scancode-toolkit/pull/3220 and https://github.com/nexB/saneyaml/issues/10, so this was a workaround, and yes this is stripped while loaded, and note that these texts are not the API, the API and reference for these texts are in the licenseDB dump or the licensedb website, for example: https://scancode-licensedb.aboutcode.org/apache-2.0.LICENSE
So this is fine to have. If you want we can track this separately also.